Kudu.Core/Scripts/postdeployment.cmd

```diff
@@ -3,10 +3,16 @@
 setlocal enabledelayedexpansion
 
 IF EXIST %POST_DEPLOYMENT_ACTIONS_DIR% (
-  FOR /F %%A IN ('dir /s /b %POST_DEPLOYMENT_ACTIONS_DIR%\*.cmd %POST_DEPLOYMENT_ACTIONS_DIR%\*.bat') DO (
+  FOR /F %%A IN ('dir /s /b %POST_DEPLOYMENT_ACTIONS_DIR%\*.cmd %POST_DEPLOYMENT_ACTIONS_DIR%\*.bat %POST_DEPLOYMENT_ACTIONS_DIR%\*.ps1') DO (
     set CURRENT_SCRIPT_FILE=%%A
-    echo Executing - '!CURRENT_SCRIPT_FILE!'
-    call !CURRENT_SCRIPT_FILE!
+    :: Get last four chars and check if file end with '.ps1' (powershell file)
+    IF /I "!CURRENT_SCRIPT_FILE:~-4!"==".ps1" (
+        echo Executing PS - '!CURRENT_SCRIPT_FILE!'
+        call PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted "!CURRENT_SCRIPT_FILE!"
```

Comment:
We seem to use `RemoteSigned` in a couple other places instead of `unrestricted`